packages/cdktf/lib/testing/matchers.ts

```diff
@@ -29,8 +31,13 @@ export function returnMatcherToJest(
   };
 }
 
-// All expected properties are matched and considered equal if
-// There can be more properties in the received object than in the expected object while still returning true
+/**
+ * Compares expected and recieved. All expected properties are matched and considered equal if
```

Comment:
```suggestion
 * Compares expected and received. All expected properties are matched and considered equal if
```

Reply:
also: the sentence just stops after "if" 😅

Reply:
The sentence goes on in the next line